doc/changes/changes_6.1.0.md

```diff
@@ -4,7 +4,7 @@ Code name: t.b.d.
 
 ## Summary
 
-t.b.d.
+This releases adds support for Pandas 2 pyarrow dtype columns for emitting data from Python UDFs. Furthermore, it fixes a silent data corruption when emitting dateframes with float16 dtype columns from Python UDFs.
```

Comment:
```suggestion
This releases adds support for Pandas 2 pyarrow dtype columns for emitting dataframes from Python UDFs. Furthermore, it fixes a silent data corruption when emitting dateframes with float16 dtype columns from Python UDFs.
```

Reply:
Also, should we document somewhere which Python/Numpy datatypes we support as input/output in datanframes?

Reply:
Ah, I see you did this already. Good job! :)